modules/500-upmeter/openapi/conversions/v3.yaml

```
@@ -0,0 +1,4 @@
+version: 3
+conversions:
+  - if .smokeMini.ingressClass != null then .ingressClass = .smokeMini.ingressClass end
```

```suggestion
  - if .smokeMini.ingressClass != null then .ingressClass = .smokeMini.ingressClass end
```
it seems like unneeded action because it was applied only for some mini. Making it the module ingress class may lead to unexpected side effects like unavailability of status or ui pages beacuse of the ingress class change.

My advise will be just to drop the smokeMini.ingressClass value in the new version of the config.